public/javascripts/SVLabel/src/SVLabel/Keyboard.js

```diff
@@ -1,10 +1,53 @@
+window.addEventListener("keydown", function(e) {
```

Comment:
Why do you need a new event listener? Look at line 73 and 74. We listeners are already attached to `document`.